src/tests/fuzz/fuzz_decodebase64.c

```diff
@@ -45,8 +39,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
     if (size < BLK_SIZE)
         return 0;
 
-    uint32_t dest_size = (uint32_t)(data[0] << 8) | (uint32_t)(data[1]);
-    Base64FuzzTest(data + BLK_SIZE, size - BLK_SIZE, dest_size);
```

Comment:
Why do you remove the randomized `dest_size` out of fuzzing ?

Reply:
Because the callers are either
- no longer sending a destination size to the decoder and the decoder is calculating the expected destination size based on the input len (mime)
- the decoder is sending the entire input length as destination size so it is always ample.

wdyt?

Reply:
> the decoder is sending the entire input length as destination size so it is always ample.

The decoder could optimize by having some rust function computing the maximum needed size